src/components/Infopic/Infopic.tsx

```diff
@@ -98,29 +98,31 @@ const InfoPic = ({
 }: InfopicProps) => {
   return (
     <HomepageSectionWrapper sectionIndex={sectionIndex}>
-      {sectionIndex && sectionIndex % 2 === 0 ? (
-        <InfopicContentWrapper shouldShowMobile={false}>
-          <TextComponent
-            title={title}
-            subtitle={subtitle}
-            description={description}
-            buttonLabel={button}
-            buttonUrl={url}
-          />
-          <ImageComponent image={image} alt={alt} />
-        </InfopicContentWrapper>
-      ) : (
-        <InfopicContentWrapper shouldShowMobile={false}>
-          <ImageComponent image={image} alt={alt} />
-          <TextComponent
-            title={title}
-            subtitle={subtitle}
-            description={description}
-            buttonLabel={button}
-            buttonUrl={url}
-          />
-        </InfopicContentWrapper>
-      )}
+      <InfopicContentWrapper shouldShowMobile={false}>
+        {(sectionIndex ?? 0) % 2 === 0 ? (
```

Comment:
thought: this binds the left right appearance of the picture with the grey background. I was thinking perhaps we can have a separate prop to define the left and right alignment of the picture?

If this is the current behaviour then let's park this for now and we can follow up with design.

Reply:
[52f5b1f](https://github.com/isomerpages/isomer-components/pull/5/commits/52f5b1ffb114c5ab30ca7c18affd10b08ab3b81c)

oh yea you're right!
current behaviour is just that it alternates from left to right, whereas gray or white depends on section id

Reply:
lol a little easter egg for you because I thought I understood the homepage.html template correctly but something was different on CMS


https://github.com/isomerpages/isomer-components/assets/77665182/39122d85-96d1-4fa4-9910-8447af835ce7

Reply:
LOL damn